data/species_list.txt

```diff
@@ -0,0 +1,3 @@
+homo_sapiens
```

Comment:
Is this file for testing?

Reply:
This is an example file with species for which we want to run the pipeline.